packages/text-bitmap/src/BitmapText.ts

```diff
@@ -301,6 +308,7 @@ export class BitmapText extends Container
             charRenderData.charCode = charCode;
             charRenderData.position.x = pos.x + charData.xOffset + (this._letterSpacing / 2);
             charRenderData.position.y = pos.y + charData.yOffset;
+            charRenderData.prevSpaces = this._align === 'justify' ? spaceCount : 1;
```

Comment:
Following up on the other suggestion, keep `prevSpaces` consistent across alignements:

```suggestion
            charRenderData.prevSpaces = spaceCount;
```

Reply:
Hi @SukantPal ,
Indeed you are correct. I was just thinking that my way might be a bit more efficient. But on second thought, the added comparison is compensated by one less comparison in your second fix, so I will update the PR with your changes.